venafi/resource_venafi_certificate_test.go

```diff
@@ -264,11 +321,38 @@ output "certificate" {
 output "private_key" {
 	value = "${venafi_certificate.cloud_certificate.private_key_pem}"
 }`
+	cloudCsrServiceConfigImportCreate = `
+%s
+resource "venafi_certificate" "cloud_certificate" {
+	common_name = "%s"
+	%s
+	key_password = "%s"
+	expiration_window = %d
+	csr_origin = "service"
+}
+output "certificate" {
+	value = "${venafi_certificate.cloud_certificate.certificate}"
+}
+output "private_key" {
+	value = "${venafi_certificate.cloud_certificate.private_key_pem}"
+}
+output "cert_id" {
+	value = "${venafi_certificate.cloud_certificate.id}"
+}`
+)
+
+const (
+	issuer_hint        = "MICROSOFT"
+	valid_days         = 30
+	vaas_id_path       = "/test_files/vaas_id"
+	custom_fields_path = "/test_files/custom_fields.json"
 )
 
+type KeyFormat int
+
 const (
-	issuer_hint = "MICROSOFT"
-	valid_days  = 30
+	expectedPrivKeyPKCS1 KeyFormat = iota
```

Comment:
could we put this on just one constant list?, for example on the constant declared on line: 344